src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -86,6 +87,23 @@ public SpanContext Extract<T>(T headers)
             return new SpanContext(traceId, parentId, samplingPriority, null, origin);
         }
 
+        public IEnumerable<KeyValuePair<string, string>> ExtractHeaderTags<T>(T headers, IEnumerable<KeyValuePair<string, string>> headerTags)
+            where T : IHeadersCollection
+        {
+            Dictionary<string, string> tagsFromHeaders = new Dictionary<string, string>();
+
+            foreach (KeyValuePair<string, string> headerNameToTagName in headerTags)
```

Comment:
Consider renaming the `headerTags` parameter to `headerToTagMap` or something else that makes it clear that it's a map from header names to tag names. It wasn't clear to me until I saw this `headerNameToTagName`.

Reply:
Resolved in d3b30d3d5bd5fa8fe2de77babebdc3c44f64fcee